src/Core/Checkout/Order/OrderEntity.php

```diff
@@ -369,12 +369,12 @@ public function setLanguageId(string $languageId): void
         $this->languageId = $languageId;
     }
 
-    public function getLanguage(): LanguageEntity
+    public function getLanguage(): ?LanguageEntity
     {
         return $this->language;
     }
 
-    public function setLanguage(LanguageEntity $language): void
+    public function setLanguage(?LanguageEntity $language): void
```

Comment:
Hi @JoshuaBehrens, this change is unnecessary because we can have $language nullable in the first place but prevent setting it to null.

Reply:
Is it really worth fighting for the type on a setter, when most of the core code uses `->assign(['language' => ...])`?

Reply:
Make sense to me 😄